its/projects/ExternalIssues.CS/ExternalIssues.CS/Foo.cs

```diff
@@ -1,24 +1,4 @@
-﻿/*
- * SonarScanner for MSBuild
- * Copyright (C) 2016-2023 SonarSource SA
```

Comment:
This removal is intentional, because it's IT test source => it doesn't make sense to waste time maintaining this